doc/introduction/framework/dcc.rst

```diff
@@ -0,0 +1,52 @@
+..
+    :copyright: Copyright (c) 2022 ftrack
+
+.. _introduction/framework/dcc:
+
+*********************
+DCC integration layer
+*********************
+
+Repository: https://github.com/ftrackhq/ftrack-connect-pipeline-<host_type>.git
```

Comment:
@henriknorin-ftrack maybe host_type is something we can try to reference and link from the glossary ?